package/MDAnalysis/analysis/align.py

```diff
@@ -943,6 +943,11 @@ def _conclude(self):
         self._writer.close()
         if not self._verbose:
             logging.disable(logging.NOTSET)
+    
+    def _setup_scheduler(self, scheduler, n_workers):
```

Comment:
I am not sure this is the best way to mark whether one analysis is **currently** parallelizable.

Previously, when I dealt with `Transformation`, I added a parameter `parallelizable` so that every subclass can decide whether to raise an error/override to use serial code path. https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/transformations/base.py#L108

Maybe it's a better solution?